src/Appwrite/Auth/Validator/MockNumber.php

```diff
@@ -46,7 +46,7 @@ public function isValid($value): bool
             return false;
         }
 
-        $otp = new Text(6, 6, Text::NUMBERS);
+        $otp = new Text(6, 6, Validator\Text::NUMBERS);
```

Comment:
```suggestion
        $otp = new Text(6, 6, Text::NUMBERS);
```
Can be removed right?